wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/RdfConverter.java

```diff
@@ -363,6 +364,24 @@ void writeTermTriples(Resource subject, URI predicate,
 		}
 	}
 
+	/**
+	 * Writes a triple for the {@link StatementRank} of a {@link Statement} to
+	 * the dump.
+	 * 
+	 * @param subject
+	 * @param rank
+	 */
+	void writeStatementRankTriple(Resource subject, StatementRank rank) {
+		String rankRepresentation = rankToRdfRepresentation(rank);
+		try {
+			this.rdfWriter.writeTripleUriObject(subject, RdfWriter.WB_RANK,
+					rankRepresentation);
+		} catch (RDFHandlerException e) {
+			logger.warn("Could not write rank " + rank.toString()
+					+ " of statement " + subject.toString());
```

Comment:
Better just throw new RuntimeException(e.toString(), e); as we do in all other cases where we catch this. Should almost never happen and if it does we should better escalate the issue instead of swallowing it silently. Most likely, the file has been closed or the disk is full, so we won't be able to recover from this anyway.